src/Interfaces/hamiltonians.jl

```diff
@@ -358,15 +371,15 @@ by `ham`. Alternatively, pass as argument an iterator over the accessible matrix
 
 Part of the [`AbstractHamiltonian`](@ref) interface.
 """
-function random_offdiagonal(offdiagonals::AbstractVector)
-    nl = length(offdiagonals) # check how many sites we could get_offdiagonal to
+function random_element(column::AbstractVector)
```

Comment:
I'm a bit confused by this name change. Does it still always give a random offdiagonal or not? Is the name change due to the fact that some diagonal elements might be hiding in the "offdiagonals"?